src/libs/SidebarUtils.ts

```diff
@@ -113,13 +122,18 @@ function getOrderedReportIDs(
     const isInDefaultMode = !isInGSDMode;
     const allReportsDictValues = Object.values(allReports);
 
+    const reportIDsWithViolations: Record<string, boolean> = {};
```

Comment:
I would use a `Set` instead of an `{ }` given that we only care if the reportID has a violation